From 797ed01eaecc64ee90da799ba671dafb3b5a20af Mon Sep 17 00:00:00 2001
From: WangYuli <wangyuli@uniontech.com>
Date: Tue, 13 Aug 2024 11:27:36 +0800
Subject: [PATCH 190/360] x86: config: Increase maximum number of CPUs to 512

The default number is 64.
At a cloud environment, you may be assigned 128 or more
CPUs so that 64 is not enough.

Reported-by: Zhao Ou <zhaoou@uniontech.com>
Signed-off-by: WangYuli <wangyuli@uniontech.com>
---
 arch/x86/configs/deepin_x86_desktop_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/configs/deepin_x86_desktop_defconfig b/arch/x86/configs/deepin_x86_desktop_defconfig
index 5d59afaaafab..b5520b60edd4 100644
--- a/arch/x86/configs/deepin_x86_desktop_defconfig
+++ b/arch/x86/configs/deepin_x86_desktop_defconfig
@@ -68,6 +68,7 @@ CONFIG_JAILHOUSE_GUEST=y
 CONFIG_ACRN_GUEST=y
 CONFIG_PROCESSOR_SELECT=y
 CONFIG_GART_IOMMU=y
+CONFIG_NR_CPUS=512
 CONFIG_X86_REROUTE_FOR_BROKEN_BOOT_IRQS=y
 CONFIG_X86_MCELOG_LEGACY=y
 CONFIG_PERF_EVENTS_INTEL_RAPL=m
-- 
2.20.1

